modules/ROOT/pages/security/ssl-framework.adoc

```diff
@@ -823,24 +834,41 @@ For more details on securing the comunication between the cluster servers, see x
 [[ssl-backup-config]]
 === Configure SSL for backup communication
 
-In a single instance, by default the backup communication happens on port `6362`.
-In a cluster topology, it is possible to take a backup from any server, and each server has two configurable ports capable of serving a backup.
-These ports are configured by `dbms.backup.listen.address` (port `6362`) and `server.cluster.listen_address` (port `6000`) respectively.
-If the intra-cluster encryption is enabled and the backup communication is using port `6000`, then your communication channels are already encrypted.
-The following steps assumes that your backup is set up on a different port.
+In a single instance, the backup communication happens on port `6362` by default. +
+In a cluster topology, it is possible to take a backup from any server, and each server has two configurable ports capable of serving a backup:
+
+* `dbms.backup.listen.address` (port `6362`)
+* `server.cluster.listen_address` (port `6000`)
+
+If the <<ssl-cluster-config, intra-cluster encryption>> is enabled and the backup communication uses port `6000`, then the communication channels are already encrypted.
 
-To set up SSL for backup communication, create the folder structure and place the key file and the certificates under those.
-Then, you need to configure the SSL backup policies in the _neo4j.conf_ file.
+However, if your backup communication uses a different port, you need to enable SSL for it by creating a separate SSL policy.
 
-. Set up the _backup_ folder under _certificates_.
-.. Create a directory _backup_ under _<NEO4J_HOME>/certificates_ folder:
+When setting up SSL for backup communication, you can choose between two options for certificates:
+
+* self-signed certificates -- This is typically the case where the system is fully internal (not internet-facing), you control both ends of the connection, and the distribution of certificates can be automated and secured.
```

Comment:
Self-Signed Certs vs CA signed Certs isn't really about a service being internet facing or not.  Many organizations will run their own in-house Certificate Authority (CA) to mange non-internet facing services in a secure manner.  

CA-signed certificates are more secure because a known trusted authority verifies the identity of the server, ensuring authenticity and preventing impersonation. Self-signed certificates skip this verification, so they can’t be trusted by clients and are vulnerable to man-in-the-middle attacks.